src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -69,6 +70,13 @@ public static boolean isCloudStorageUrl(final java.nio.file.Path path) {
         return ("" + path.toUri().getScheme() + "://").equals(GCS_PREFIX);
     }
 
+    /**
+     * Returns true if the given path is a HDFS (Hadoop filesystem) URL.
+     */
+    public static boolean isHadoopUrl(GATKPathSpecifier pathSpecifier) {
+        return pathSpecifier == null ? false : pathSpecifier.getURI().getScheme().equals(HDFS_SCHEME);
```

Comment:
Perhaps this sort of method (querying the type of URL) should live in `GATKPathSpecifier` itself.

Reply:
Yes. I'll migrate them there with tests in the next PR.